.vscode/settings.json

```diff
@@ -1,63 +1,61 @@
 {
   // Angular
   "vsicons.presets.angular": true,
-
   // CSS
   "css.validate": false,
   "scss.validate": false,
-
   // Default Formatting
   "editor.tabSize": 4,
   "editor.detectIndentation": false,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": false,
-
   // Per Language Formatting
   // Keep this up to date with .prettierignore
   // Run Prettier first, followed by ESLint
   "[javascript]": {
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": [
-        "source.formatDocument",
-        "source.fixAll.eslint"
+      "source.formatDocument",
+      "source.fixAll"
```

Comment:
Removed this and the whitespace changes (introduced by a local VSCode extension; sorry!). Also adding VSCode Icons as a recommended extension, since we already have a setting for it (Line 3).